core/testing/go.mod

```diff
@@ -7,5 +7,9 @@ replace cosmossdk.io/core => ../
 require (
 	cosmossdk.io/core v1.0.0
 	github.com/golang/mock v1.6.0
+	github.com/spf13/cast v1.7.0
+	github.com/syndtr/goleveldb v1.0.1-0.20220721030215-126854af5e6d
```

Comment:
do we need this dep?

Reply:
i am also curious if we should have a `goleveldb` dep in core/testing, it seems a bit off

Reply:
Can we remove in this pr?  Ideally we don't merge to main things we will revert later on